src/blocks/helpers/helper-functions.js

```diff
@@ -289,6 +289,50 @@ export const hex2rgba = ( color, alpha = 100 ) => {
 	return `rgba(${r},${g},${b},${alpha / 100})`;
 };
 
+/**
+ * Converts HEX colors to HSL.
+ *
+ * @param color
+ * @param asArray
+ * @returns {string|array}
+ */
+export const rgb2hsl = ( color, asArray = true ) => {
```

Comment:
I recommend moving this to `frontend-helper-functions.js` since it is used only on the front end.

That file is a dependency-free file, it will be easier to make a unit test for it later.